linkerd/proxy/tap/src/accept.rs

```diff
@@ -58,9 +58,9 @@ impl AcceptPermittedClients {
 }
 
 impl Service<Connection> for AcceptPermittedClients {
-    type Response = ();
+    type Response = Box<dyn Future<Item = (), Error = Error> + Send + 'static>;
```

Comment:
This looks like it's always a `ServeFuture`, so we don't need to box it:
```suggestion
    type Response = ServeFuture;
```